docs/Configuration.md

```diff
@@ -44,6 +44,20 @@ These options let you control Jest's behavior in your `package.json` file. The
 Jest philosophy is to work great by default, but sometimes you just need more
 configuration power.
 
+### Defaults
+
+You can retrieve Jest's default options to expand them if needed:
+
+```js
+// jest.config.js
+const defaults = require('jest-config').defaults;
+module.exports = {
+  // ...
+  moduleFileExtensions: [...defaults.moduleFileExtensions, 'ts', 'tsx'],
+  // ...
+};
+```
+
 <AUTOGENERATED_TABLE_OF_CONTENTS>
```

Comment:
The new heading should be below the ToC, shouldn't it?

Reply:
@SimenB tried, less readable

![screen shot 2018-04-09 at 17 07 04](https://user-images.githubusercontent.com/643434/38505847-7b82639c-3c18-11e8-9186-9858ff003820.png)